letsencrypt_transip_nl/README.rst

```diff
@@ -0,0 +1,86 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==============================
+LetsEncrypt transip.nl support
+==============================
+
+This module was written to extend the functionality of letsencrypt module
+to support transip.nl DNS challenges. 
+
+Upon receiving a challenge of type DNS, this module will automatically create
+the relevant DNS record on your domain.
+
+Installation
+============
+
+To install this module, you need to:
+
+Follow the usuall installation procedure for a module in your Odoo version.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+Go to Settings -> General Settings and fill the required fields under
+LetsEncrypt.
+
+Those fields are:
+    DNS Provider: Set this to TransIP
+    Login: Set this to the username you use to login to transip.nl
+    Key: Set this to the private key given to you by tranip.nl in order to use their APIs.
+    URL: Set the URL of the APIs. Usually api.transip.nl
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. go to ...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+:alt: Try me on Runbot
+:target: https://runbot.odoo-community.org/runbot/149/10.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/server-tools/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* George Daramouskas <gdaramouskas@therp.nl>  
+
+Do not contact contributors directly about help with questions or problems concerning this addon, but use the `community mailing list <mailto:community@mail.odoo.com>`_ or the `appropriate specialized mailinglist <https://odoo-community.org/groups>`_ for help, and the bug tracker linked in `Bug Tracker`_ above for technical issues.
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
```

Comment:
for new modules, use [split readme files](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)

Reply:
Not sure if I understand that readme file. Basically the README.rst that exists on the module's top directory is now removed and in its place we have a readme folder with various rst files as shown in the `maintainer-tools`?

Reply:
yes, in a `readme` subfolder. And then you can call https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py (this needs to be installed to generate the correct callables) to actually generate the readme to be sure it looks the way you want